### ISSUE TITLE
std::(w)string support in mixed projects

### ISSUE THREAD

**Issue by muffmolch** [fmtlib/fmt#2955]
I cannot get a good solution for me from the other questions similar to my one. 
We switched from fmt v4.1 to 8.1.1.
We are using VS2019, c++14, predefs: FMT_HEADER_ONLY. Following snippet 
`auto s1 = fmt::format(_T(">{:d}"), 7);`
results in:
`error C2664: 'std::string fmt::v8::format<int>(fmt::v8::basic_format_string<char,int>,int &&)': cannot convert argument 1 from 'const wchar_t [6]' to 'fmt::v8::basic_format_string<char,int>'`

Thus, cause this file is uses for ANSI and UNICODE projects, we need an expression that works for both. In fmt 4.1 it works like a charm. Is there a workaorund or even better solution? maybe fmt::tformat?

**Comment by muffmolch** [comment]
Or is `#include <fmt/xchar.h>` the correct and long time support solution for this problem?

**Comment by vitaut** [comment]
> Or is #include <fmt/xchar.h> the correct and long time support solution for this problem?

Yes, you should include `fmt/xchar.h` for `wchar_t` support per documentation: https://fmt.dev/latest/api.html#xchar-api.
